jest.config.mjs

```diff
@@ -1,9 +1,8 @@
 export default {
   verbose: true,
-  testMatch: ['<rootDir>/test/**/*.?js'],
+  testMatch: ["<rootDir>/test/**/*.?js"],
   transform: {},
   moduleFileExtensions: ["js", "jsx", "mjs"],
   testPathIgnorePatterns: ["node_modules", "__mocks__", "support"],
-  roots: ["<rootDir>/src/", "<rootDir>/test/"]
+  roots: ["<rootDir>/src/", "<rootDir>/test/"],
```

Comment:
**Annotation:**
I switched to a new IDE that does a better job formatting code, so changes here are just `prettier` formatting being applied.